xl2times/utils.py

```diff
@@ -355,43 +381,32 @@ def get_logger(log_name: str = default_log_name, log_dir: str = ".") -> loguru.L
         Name of the log. Corresponding log file will be called {log_name}.log. (Default value = default_log_name)
     log_dir
         Directory to write the log file to. Default is the current working directory.
-
-    Returns
-    -------
-    Logger
-        A configured loguru logger.
     """
-    from loguru import logger
-
-    # set global log level via env var.  Set to INFO if not already set.
-    if os.getenv("LOGURU_LEVEL") is None:
-        os.environ["LOGURU_LEVEL"] = "INFO"
-
-    log_conf = {
-        "handlers": [
-            {
-                "sink": sys.stdout,
-                "diagnose": True,
-                "format": "<green>{time:YYYY-MM-DD HH:mm:ss.SSS}</green> | <level>{level: <8}</level> : <level>{"
-                "message}</level> (<cyan>{name}:{"
-                'thread.name}:pid-{process}</cyan> "<cyan>{'
-                'file.path}</cyan>:<cyan>{line}</cyan>")',
-            },
-            {
-                "sink": f"{log_dir}/{log_name}.log",
-                "enqueue": True,
-                "mode": "a+",
-                "level": "DEBUG",
-                "colorize": False,
-                "serialize": False,
-                "diagnose": False,
-                "rotation": "20 MB",
-                "compression": "zip",
-            },
-        ],
-    }
-    logger.configure(**log_conf)
-    return logger
+    log_level = set_log_level(level)
+
+    if level is not None and level > 0:
+        format = '<green>{time:YYYY-MM-DD HH:mm:ss.SSS}</green> | <level>{level: >8}</level> : <level>{message}</level> (<cyan>{name}:{thread.name}:pid-{process}</cyan> "<cyan>{file.path}</cyan>:<cyan>{line}</cyan>")'
+    else:
+        format = "<green>{time:YYYY-MM-DD HH:mm:ss.SSS}</green> | <level>{level: >8}</level> : <level>{message}</level>"
+
+    logger.remove()
+    logger.add(
```

Comment:
The docs seem to suggest this is the way to setup loguru: https://loguru.readthedocs.io/en/stable/resources/troubleshooting.html#how-do-i-create-and-configure-a-logger